package.json

```diff
@@ -0,0 +1,11 @@
+{
+  "name": "mayflower",
+  "version": "1.0.0",
+  "repository": "https://github.com/massgov/mayflower.git",
+  "license": "UNLICENSED",
+  "dependencies": {
```

Comment:
We should make these `devDependencies` instead of dependencies. No functional difference in this case, but it's good practice.